node/validator.go

```diff
@@ -82,6 +84,12 @@ type validator struct {
 }
 
 func (n *validator) SendBid(ctx context.Context, args types.BidArgs) (common.Hash, error) {
+	bidFeeCeil := big.NewInt(int64(n.cfg.BidFeeCeil))
+
+	if args.RawBid.BuilderFee.Cmp(bidFeeCeil) > 0 {
+		return common.Hash{}, fmt.Errorf("bid fee exceeds the ceiling %v", n.cfg.BidFeeCeil)
```

Comment:
shall we do this check earlier, it seems a bit later just before really send the Bid to validator

Reply:
done